translations/component-properties/tracked-vs-cp/es.yaml

```
@@ -0,0 +1,4 @@
+title: Utilice @tracked y Getters en lugar de propiedades computadas
+description: >
+  Etiquete las propiedades que se deben rastrear.
+  Los Getters se actualizará automáticamente.
```

I may be wrong here. Would the verb end in `-án` since "will update" is future tense, plural?

```suggestion
  Los Getters se actualizarán automáticamente.
```